lib/versions-complete.xml

```diff
@@ -4436,6 +4436,18 @@
 					</license>
 				</licenses>
 			</library>
+			<library>
+				<file-name>com.liferay.portal.template.velocity.jar!commons-collections.jar</file-name>
+				<version>3.2.2</version>
```

Comment:
@JanisZhang Hi Janis, when committing the change generated by `build-lib-versions`, please only commit the parts relevant to the PR. For example, in this commit, please exclude the parts related to "commons-collections" and "commons-beanutils".

Reply:
@dantewang  Hi Dante, also `commons-codec.jar.`